tier2/hooks/post_gen_project.py

```diff
@@ -30,8 +31,52 @@ def addTopic():
     ]
     subprocess.call(gh_cli_command)
 
+def addMaintainer():
+    maintainers = []
+    add_maintainer = True
+    while add_maintainer:
+        maintainer = {}
+        maintainer["role"] = input("Maintainer's Role (Reviewer, Approver, Maintainer): ").strip()
+        maintainer["name"] = input("Maintainer's Name: ").strip()
+        maintainer["github_username"] = input("Maintainer's GitHub Username: ").strip()
+        maintainer["affiliation"] = input("Maintainer's Affiliation (DSAC, CCSQ, CMMI, etc...): ").strip()
+        maintainers.append(maintainer)
+
+        while True:
+            add_maintainer_input = input("Would you like to add another maintainer? (yes/no): ").strip().lower()
+            if add_maintainer_input in ("yes", "y", "no", "n"):
+                # If input is "yes" or "y", then add_maintainer is True and main loop continues, vise versa
+                add_maintainer = add_maintainer_input in ("yes", "y")
+                break
+            else:
+                print("\nInvalid response, please respond with: 'yes', 'y', 'no', or 'n'")
+
+    maintainers_table = ""
+    for maintainer in maintainers:
+        maintainers_table += f"| {maintainer["role"]} | {maintainer["name"]}| {maintainer["github_username"]} | {maintainer["affiliation"]} |\n"
+
+    proj_name = "{{ cookiecutter.project_name }}"
+    maintainers_file_path = f"../{proj_name}/MAINTAINERS.md"
+
+    with open(maintainers_file_path, "r") as f:
+        lines = f.readlines()
+
+    with open(maintainers_file_path, "w") as f:
+        in_table = False
+        for line in lines:
+            if "| {role} | {names} | {github usernames} | {affiliations}|" in line:
+                in_table = True
+                f.write(maintainers_table)  # Replace placeholder line with new table of maintainers
+            elif in_table and line.strip() == "":
```

Comment:
You could probably just get rid of the `in_table` variable and replace the `elif` and `if` with just an `else`